docs/contributing.rst

```diff
@@ -133,6 +140,31 @@ To run the skipped tests for the whole terminal session, execute::
 
 and then run the tests as usual.
 
+Running tests with large files
+------------------------------
+
+To run the tests with large input or output files, ensure you have the
+`Git Large File Storage`_ extension installed and run the tests as usual.
+
+Adding tests with large files
+-----------------------------
+
+If a test file is larger than 1 MiB, then put it in the
+``resolwe_bio/tests/processes/files/large/`` directory. Git Large File Storage
+(LFS) extension will automatically pick it up and treat it appropriately.
+
+To ensure contributors without Git LFS or users using the source distribution
+can smoothly run the tests, decorate the tests using large files with the
+following::
+
+    @unittest.skipUnless(*check_large_files(<large-file1>, <large-file2>, ...))
```

Comment:
@mstajdohar, thanks for the comment. I agree we could make this friendlier.

I don't like `@skip_large_files` since the meaning of the decorator is "Skip the test unless the given large files are present".

What about `@skipUnlessLargeFilesPresent(<large-file1>, <large-file2>, ...)` or `@skipIfLargeFilesNotPresent(<large-file1>, <large-file2>, ...)`?

@dblenkus, what are your preferences?